app/views/idv/in_person/ready_to_verify/show.html.erb

```diff
@@ -111,3 +111,7 @@
     </address>
   </section>
 <% end %>
+
+<%= render PageFooterComponent.new(class: 'padding-top-4') do %>
```

Comment:
Curious why we're adding an ad hoc padding value. Is the default not sufficient?

Reply:
That's a good question. I added it to match the designs but now I've noticed that the other cancel buttons in the application have consistent spacing that doesn't match the designs. I'll remove this bit to keep the current pattern and call out this difference to the designer who reviews this PR